pypeit/specobj.py

```diff
@@ -137,7 +138,8 @@ class SpecObj(datamodel.DataContainer):
                                             'used for this extraction'),
                  'BOX_CHI2': dict(otype=np.ndarray, atype=float,
                                   descr='Reduced chi2 of the model fit for this spectral pixel'),
-                 'BOX_RADIUS': dict(otype=float, descr='Size of boxcar radius (pixels)'),
+                 'BOX_R_PIX': dict(otype=float, descr='Size of boxcar radius (pixels)'),
```

Comment:
Am trying to decide if this will make us truly backwards incompatible.  In the context of displaying, not as much for coadding.     Can you test whether this branch can display old `spec1d` files with `pypeit_show_1dspec`?   I assume not, in part because we never had a `--try_old` option.

Seems like we might want to add one for this...